### ISSUE TITLE
App not installed

### ISSUE THREAD

**Issue by PemmasaniRajesh** [glcas/faceDemo#1]
I'm not able to install this apk in few devices like Samsung Galaxy A10, Moto C plus, Redmi 8A. I'm thinking it's because of ndk libraries that supports only few mobile cpu architecture. 

Is there any solution for this ?

**Comment by glcas** [comment]
Sorry, I only built for arm64-v8a😂. 
However, the framework I used supports armeabi-v7a. You can download their library [here](https://github.com/Tencent/ncnn/releases) and add to this project.

**Comment by PemmasaniRajesh** [comment]
Thanks for the update. I've added the armeabi-v7a library from Tencent/ncnn and it's working fine in Samsung devices. But in Redmi Note8 it's getting stuck while installing. I'm sharing the device hardware info below. 

![MobileInfo](https://user-images.githubusercontent.com/10286949/94786061-1dae6400-03ee-11eb-83be-bd59678e9e11.jpeg)

I've added only the armeabi-v7a library. Do I've to update the arm64-v8a and include folder files also ?
